src/main/java/club/sk1er/hytilities/util/locraw/LocrawUtil.java

```diff
@@ -55,6 +57,13 @@ public void onWorldLoad(WorldEvent.Load event) {
         tick = 0;
     }
 
+    public String onMessageSend(@NotNull String message){
```

Comment:
use the dual module system to make this a ChatSendModule compliant sort of thing